src/libstore/legacy-ssh-store.cc

```diff
@@ -74,7 +74,7 @@ struct LegacySSHStore : public virtual LegacySSHStoreConfig, public virtual Stor
     {
         auto conn = make_ref<Connection>();
         conn->sshConn = master.startCommand(
-            fmt("%s --serve --write", remoteProgram)
+            fmt("PATH=\"$PATH:$HOME/.nix-profile/bin:/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin\" %s --serve --write", remoteProgram)
```

Comment:
I don't think we need /run/current system here. That would only work if the remote machine is nixos and that has nix* in PATH.

Reply:
Ping @balsoft ?

Reply:
Sure